helpers/ContentLoader.php

```diff
@@ -43,7 +43,13 @@ public function __construct() {
     public function update() {
         $sourcesDao = new \daos\Sources();
         foreach($sourcesDao->getByLastUpdate() as $source) {
-            $this->fetch($source);
+        	  $last = $source['lastupdate'];
+        	  $wait = $source['waitperiod'];
+        	  if (($last + $wait) < time()){
+	            $this->fetch($source);
+//            } else {
```

Comment:
It would make sense to log it, I am not sure where should the echoed output appear.